src/metabase/metabot.clj

```diff
@@ -0,0 +1,70 @@
+(ns metabase.metabot
+  "The core metabot namespace. Consists primarily of functions named infer-X,
+  where X is the thing we want to extract from the bot response."
+  (:require
+   [metabase.lib.native :as lib-native]
+   [metabase.metabot.client :as metabot-client]
+   [metabase.metabot.settings :as metabot-settings]
+   [metabase.metabot.util :as metabot-util]
+   [metabase.util.log :as log]))
```

Comment:
In future iterations there may be an abstraction that emerges along the lines of build-context -> generate-prompt -> run-inferencer -> postprocess-result. ATM I'd like at least one more example to work with to see how this will shake out.